apis/types/task_create_request.go

```diff
@@ -6,7 +6,7 @@ package types
 // Editing this file might prove futile when you re-run the swagger generate command
 
 import (
-	strfmt "github.com/go-openapi/strfmt"
+	"github.com/go-openapi/strfmt"
```

Comment:
You should not update the files under the dir `apis` . As an instead, you should  update the `swagger.yml` firstly and then run the script `./hack/generate-swagger-models.sh` to generate the code by `go-swagger`.